x-pack/plugins/uptime/public/pages/certificates.tsx

```diff
@@ -72,26 +72,35 @@ export const CertificatesPage: React.FC = () => {
   }, [dispatch, page, search, sort.direction, sort.field, lastRefresh]);
 
   const { data: certificates } = useSelector(certificatesSelector);
+  const history = useHistory();
 
   return (
     <>
       <EuiFlexGroup responsive={false} gutterSize="s">
         <EuiFlexItem grow={false} style={{ marginRight: 'auto', alignSelf: 'center' }}>
-          <Link to={OVERVIEW_ROUTE} data-test-subj="uptimeCertificatesToOverviewLink">
-            <EuiButtonEmpty size="s" color="primary" iconType="arrowLeft">
-              {labels.RETURN_TO_OVERVIEW}
-            </EuiButtonEmpty>
-          </Link>
+          <EuiButtonEmpty
+            color="primary"
+            data-test-subj="uptimeCertificatesToOverviewLink"
+            href={history.createHref({ pathname: OVERVIEW_ROUTE })}
+            iconType="arrowLeft"
+            size="s"
+          >
+            {labels.RETURN_TO_OVERVIEW}
+          </EuiButtonEmpty>
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
           <ToggleAlertFlyoutButton alertOptions={[CLIENT_ALERT_TYPES.TLS]} />
         </EuiFlexItem>
         <EuiFlexItem grow={false} style={{ alignSelf: 'center' }}>
-          <Link to={SETTINGS_ROUTE} data-test-subj="uptimeCertificatesToOverviewLink">
-            <EuiButtonEmpty size="s" color="primary" iconType="gear">
-              <EuiHideFor sizes={['xs']}> {labels.SETTINGS_ON_CERT}</EuiHideFor>
-            </EuiButtonEmpty>
-          </Link>
+          <EuiButtonEmpty
+            color="primary"
+            data-test-subj="uptimeCertificatesToOverviewLink"
+            iconType="gear"
+            href={history.createHref({ pathname: SETTINGS_ROUTE })}
+            size="s"
+          >
```

Comment:
```suggestion
            aria-label={labels.SETTINGS_ON_CERT}
          >
```

Because you're hiding the text of the button on mobile sizes, we need to provide an `aria-label` to provide a screen reader label.

Reply:
Added in https://github.com/elastic/kibana/pull/72926/commits/1c89a1f40a45771084bf7d6134994397ec1dfe44.